Teensy/src/gpio/gpio.cpp

```diff
@@ -96,7 +96,35 @@ static GyroData_s gpio_getGyro()
 
 static daq_BatteryV_t gpio_getVBat()
```

Comment:
daq_BatteryV_t is currently defined as an int32_t could you change this to float. Its defined in `cmn/daq_common.h`